src/foam/u2/view/RichChoiceView.js

```diff
@@ -328,6 +328,26 @@ foam.CLASS({
       this.onDetach(this.data$.sub(this.onDataUpdate));
       this.onDataUpdate();
 
+      // Set up an event listener on the window so we can close the dropdown
+      // when the user clicks somewhere else.
+      var containerElement;
+      const fn = function(evt) {
+        var selfRect = (document.getElementById(self.id)).getClientRects()[0];
+        var containerRect = (document.getElementById(containerElement.id)).getClientRects()[0];
+        if (
+          ! (
+              evt.clientX >= selfRect.x &&
+              evt.clientX <= selfRect.x + selfRect.width &&
+              evt.clientY >= selfRect.y &&
+              evt.clientY <= containerRect.y + containerRect.height
+            )
+        ) {
+          self.isOpen_ = false;
+        }
+      };
+      window.addEventListener('click', fn);
```

Comment:
Importing 'window' rather than using the global 'window' allows your app/view to work with multiple windows.